typed/declaration/examples.ipldsch.json

```diff
@@ -4,53 +4,51 @@
 			"kind": "map",
 			"keyType": "String",
 			"valueType": "String",
-			"valueNullable": true
+			"nullable": true,
+			"representation": {
+				"map": {}
+			}
 		},
 		"ExampleWithAnonDefns": {
 			"kind": "struct",
 			"fields": {
 				"fooField": {
-					"valueType": {
```

Comment:
as a struct, I think this should be "type" as "valueType" is for maps isn't it? coming from StructField->type